.env.example

```
@@ -40,3 +40,6 @@ ORDER_SERVICE_SCHEMA=order_schema
 NOTIFICATION_SERVICE_NAME=notification-service
 NOTIFICATION_SERVICE_PORT=19700
 NOTIFICATION_SERVICE_SCHEMA=notification_schema
+
+# TMAP
+TMAP_API_KEY=tmap-api-key
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)